lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -235,6 +239,22 @@ def call(env)
             env[:ui].info(" -- USB passthrough:   #{usb_dev.join(', ')}")
           end
 
+          @redirdevs.each do |redirdev|
+            redirdev_dev = []
```

Comment:
it will working correct only for 1 device
try
```
redirdev_dev = @redirdevs.map {|redirdev|  "type=#{redirdev[:type]}" if redirdev[:type]}
env[:ui].info(" -- Redirected Devices: #{redirdev_dev.join(', ')}")
```